flight_computer/packet_schemas/schemas/generic.json

```diff
@@ -1,10 +1,10 @@
 [
```

Comment:
what is the purpose of this generic.json file? Is it a base schema? Or just for testing?

Reply:
It's the fields we want all packets to have.